haskell-build/orb.yml

```diff
@@ -90,9 +90,13 @@ commands:
             BUILD_CABAL_VERSION=$(cabal --numeric-version)
             BUILD_GHC_VERSION=$(cat cabal.project | grep 'with-compiler:' | head -n 1 | tr -s ' ' | cut -d' ' -f2 | cut -d'-' -f2)
             if [ "$BUILD_GHC_VERSION" = "" ]; then BUILD_GHC_VERSION=$(ghc -- --numeric-version); fi
-            BUILD_PROJECT_NAME=$(cat "$BUILD_CABAL_FILE" | grep '^name:' | head -n 1 | cut -d':' -f 2 | xargs)
-            BUILD_PROJECT_VERSION=$(cat "$BUILD_CABAL_FILE" | grep -e "^version" | head -n 1 | tr -s " " | cut -d' ' -f2)
-            BUILD_EXE_NAMES=$(cat "$BUILD_CABAL_FILE" | grep 'executable ' | head -n 1 | tr -s ' ' | cut -d' ' -f2 || echo '')
+
+            if [ "$BUILD_CABAL_FILE" = ""]; then
+              BUILD_PROJECT_NAME=$(cat "$BUILD_CABAL_FILE" | grep '^name:' | head -n 1 | cut -d':' -f 2 | xargs)
+              BUILD_PROJECT_VERSION=$(cat "$BUILD_CABAL_FILE" | grep -e "^version" | head -n 1 | tr -s " " | cut -d' ' -f2)
+              BUILD_EXE_NAMES=$(cat "$BUILD_CABAL_FILE" | grep 'executable ' | head -n 1 | tr -s ' ' | cut -d' ' -f2 || echo '')
```

Comment:
How are these variables used?  Works okay with them not defined?

Reply:
They're used in the `copy-bin` command and in the *example* in the `github-release` orb.

The `BUILD_CABAL_FILE` var is actually used by the `github-release` orb, but as noted I'm leaving that fix for future work. (The `github-release` orb will fail if it's not set, so this not setting it here is merely potentially annoying rather than dangerous).